### PR TITLE
doc/start: rewrite intro paragraph

### PR DESCRIPTION
**Pull request by zdover23**
Rewrite the first paragraph in doc/start/intro.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/start/intro.rst [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
-Ceph OSD (Object Storage Daemon). The Ceph Metadata Server is also
-required when running Ceph File System clients.
+Ceph can be used to provide :term:`Ceph Object Storage` to :term:`Cloud
+Platforms` and Ceph can be used to provide :term:`Ceph Block Device` services
+to :term:`Cloud Platforms`. Ceph can be used to deploy a :term:`Ceph File
+System`.  All :term:`Ceph Storage Cluster` deployments begin with setting up
+each :term:`Ceph Node` and then setting up the network. 
+
+A Ceph Storage Cluster requires the following: at least one Ceph Monitor (and
+one Ceph Manager per monitor), and as many Ceph OSDs as there are copies of an
```
Comment (anthonyeleven) [inline doc/start/intro.rst:12]: Mgrs are not tied to mons this way, though it’s best practice.  Rook does not allow deploying more than two total.

File: doc/start/intro.rst [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+object stored on the Ceph cluster (for example, if three copies of a given
+object are stored on the Ceph cluster, then three OSDs must exist in that Ceph
```
Comment (anthonyeleven) [inline doc/start/intro.rst:14]: At least three

File: doc/start/intro.rst [hunk 3]
```diff
@@ -2,14 +2,19 @@
...
-:term:`Ceph Block Device` services to :term:`Cloud Platforms`, deploy
-a :term:`Ceph File System` or use Ceph for another purpose, all
-:term:`Ceph Storage Cluster` deployments begin with setting up each
-:term:`Ceph Node`, your network, and the Ceph Storage Cluster. A Ceph
-Storage Cluster requires at least one Ceph Monitor, Ceph Manager, and
-Ceph OSD (Object Storage Daemon). The Ceph Metadata Server is also
-required when running Ceph File System clients.
+Ceph can be used to provide :term:`Ceph Object Storage` to :term:`Cloud
+Platforms` and Ceph can be used to provide :term:`Ceph Block Device` services
+to :term:`Cloud Platforms`. Ceph can be used to deploy a :term:`Ceph File
```
Comment (anthonyeleven) [inline doc/start/intro.rst:7]: There are also direct librados clients but they are so few that we need not mention that here.

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
Factual changes

File: doc/start/intro.rst (hunk 1 above)
Comment (anthonyeleven) [inline doc/start/intro.rst:12]: At least as many OSDs

File: doc/start/intro.rst (hunk 3 above)
Comment (zdover23) [reply doc/start/intro.rst:7]: Eventually we'll have a section on custom-built librados clients. It's on a list.

Comment (anthonyeleven) [reply doc/start/intro.rst:7]: I only recall having seen mention of a total of *2* so probably anyone who needs to know already does.

File: doc/start/intro.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/start/intro.rst]: Did you take out the mgr altogether?  To be clear I was suggesting at least one mon and at least one mgr.  Their placement and numbers are not coupled in any way.

Comment (zdover23) [reply doc/start/intro.rst]: Crud. Serves me right for working while jetlagged.

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51220 - Reef backport
https://github.com/ceph/ceph/pull/51221 - Quincy backport
https://github.com/ceph/ceph/pull/51222 - Pacific backport

**Pull request merged**